cloudera/tests/conftest.py

```diff
@@ -59,3 +49,27 @@ def _response(filename):
             return json.load(f)
 
     return _response
+
+
+def get_timeseries_resource():
```

Comment:
Why did you change the previous syntax?
```
return [
  ApiTimeSeriesResponseList(
      items=[
          ApiTimeSeriesResponse(
              time_series=[
                  ApiTimeSeries(
                      data=[
                          ApiTimeSeriesData(value=49.7),
                      ],
                      metadata=ApiTimeSeriesMetadata(metric_name=metric, entity_name=category),
                  )
                  for metric in metrics
              ]
          ),
      ],
  )
  for category, metrics in TIMESERIES_METRICS.items()
]
```

Reply:
After talking offline, will keep the original syntax to keep the `side_effect` for the fixture.